### ISSUE TITLE
Add a check for text, focus and action on background color

### ISSUE THREAD

**Issue by alex-page** [govau/uikit#343]
- [ ] I’ve read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
- [ ] I’ve read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

# Description

## What happened

Currently you can input inaccessible colors.

## What I expected to happen

Inaccessible colors should have an error logged or the nearest possible a11y color found.

# Reproducing

- Module name: core
- Module version: 2.0.0

Steps to reproduce:

1. Install core
2. Make all the values you can input `#fff`
3. The response from the system should do something about this
